ddtrace/contrib/botocore/patch.py

```diff
@@ -69,8 +69,13 @@ def patched_api_call(original_func, instance, args, kwargs):
 
         result = original_func(*args, **kwargs)
 
-        span.set_tag(http.STATUS_CODE, result['ResponseMetadata']['HTTPStatusCode'])
-        span.set_tag('retry_attempts', result['ResponseMetadata']['RetryAttempts'])
+        response_meta = result['ResponseMetadata']
+        span.set_tag(http.STATUS_CODE, response_meta['HTTPStatusCode'])
+        span.set_tag('retry_attempts', response_meta['RetryAttempts'])
+
+        request_id = response_meta.get('RequestId')
```

Comment:
not sure guarding against an unset RequestId is really necessary, but saw [aiobotocore](https://github.com/DataDog/dd-trace-py/blob/master/ddtrace/contrib/aiobotocore/patch.py#L116) was cautious so figured I would be as well

Reply:
doesn't hurt